climada/hazard/centroids/centr.py

```diff
@@ -560,18 +560,31 @@ def remove_duplicate_points(self, scheduler=None):
         sel_cen = geom_wkb.drop_duplicates().index
         return self.select(sel_cen=sel_cen)
 
-    def select(self, reg_id=None, sel_cen=None):
+    def select(self, reg_id=None, extent=None, sel_cen=None):
         """Return Centroids with points in the given reg_id or within mask
 
         Parameters:
             reg_id (int): region to filter according to region_id values
-            sel_cen (np.array): 1-dim mask
+            extent (tuple): Format (min_lon, max_lon, min_lat, max_lat) tuple.
```

Comment:
You should consider renaming it `bbox` (like in many other CLIMADA modules) or `bounds` (like in rasterio and many CLIMADA modules). But make sure to use the order `(lon_min, lat_min, lon_max, lat_max)` in that case. The name `extent` with the order `(lon_min, lon_max, lat_min, lat_max)` is mainly used in matplotlib contexts, as far as I know.